src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -532,9 +532,9 @@ def _predict_with_probabilities(self, input_dataset, batch_size=None,
         dataset_size = len(dataset)
         ctx = _mxnet_utils.get_mxnet_context()
         
-        all_predicted = ['']*dataset_size
-        all_probabilities = _np.zeros((dataset_size, len(self.classes)), 
-            dtype=float)
+        # all_predicted = ['']*dataset_size
```

Comment:
Unless you have a good reason and a detailed comment explaining that reason, please remove commented out code prior to putting up a PR.

Reply:
This was fixed last night, but I'll keep this in mind. Sorry, some dead code slipped this time in the PR.